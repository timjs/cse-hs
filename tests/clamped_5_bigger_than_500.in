5
drpj(djyb(voc(cyqo(ceyo(awcb,bcuf),dfap(cyeo(edws(bwfk(dbru(cm(egvd,ctd(cnkx(bcv,bwmf),bgpm)),egrs(auwt,saa)),suv(buow(aovq,czqa(sfl(eto,cbda),aksb)),epmn)),cst(brbm,awoe)),aopa),akoh)),abar),aabl(exd,apem)),amma(cjwz(dlab(biyx(yey,azin(dtwu(bqfe(dctp(ecrq(doir,axgu),arcc(vpg(bwsb(cqab,bsvv),awwg),ojp)),dcpl(afla,aazx(drrm(ebzm(btyz(cgme(assm(beeu,astk),dbqg),ehzx),bgyx),bopd(ampg(eef(auzd,aaqq(bmee,dyoj(mcn,aimn))),aqnp(cspy(admh(bdft,dfyk(clv,dobh(dmgy,dqiv))),dljz(codv(btns(ekyf(ayvh,cjhg),cxor),buya),jf)),cbrh)),axmu(dxhx,eixp(ehwy,bvph(amkj(dgza(admn(arqk,ctuz(ejji(owd(adye,doei(bmkt,csay)),ghu(dyob(imv(yjv(aik(nmj,dkut(bumh(cxsh(birg(awin(dcxe,dxcp(ejxv,akzr)),asew),avnp),oxq),bnni)),afae(cysy,ccvy(delw(aszl,atwx),elmu(dodg,cwoa(xdc(caob(yja,ahs(bgyj,ger)),efji(bdjo(cdea,akyu),mkc)),bgwt(ccat(pse,eprv(ciqr,btz(dopq,cpfi(clyl,bchk)))),xda)))))),djyx(upo,dxyw)),dnzp(abik,hon(doji,apyg))),brni)),bsro(bddc,pme(dfnr,axtq)))),brjg),sp),eftq))))),edhh))),dkkp),axuh)),czaq),aadv),dqsw))
aovw(chnm(efcw,bxlu),bocj(qff(crfs,dqli(cvjg(por(uwk(dcpw,btpd),ecx),jka(ybv,cmmo)),cxdr(bqml(mki(ywy,drey),yzt(awbu(ceif,eqas(lzj(afqr(axdr(bdfs,bdzv),caob),hmt),aoos)),bjpb)),ayny))),bmgi(avqr(anuf,czge(dvmd(dxof,dudd),eewu)),xhd(hjc(clol(dasa,bdja),bmki(gna(bruw(auyd(djug(bokw(yyl,ahmf),zcg(dhcg(cwiu,awyg),clib(dizj(bwh(dnlr,dpoy(dke(alpl(alvv,eaot),dauh),dffv)),eetl),elza))),dkzj(egtp(bktb,elcw),chbx)),acpk),amko(bgoi,dnhg)),eks)),cotn(cobb,eepu(dzaj(njo(eppv(ujg(utq(dojx(dqws(cveo,tcw(ckpf,cvcg(slo(dkum,aiot),pfh))),cpnd),txb),bzyc(ekr(dfdi(arww,ejnw),boay(bcff(bela,fzp(enha,bfrx(aecz(aaao,adyp(nlw,bmqz)),egjz(dtst(dkkd,agtq(akbx,cyh(ifc(cvzi,edjk(cljk(chzb,aksq),brqs(dlfa(adfr(elv(clp(hdg,epmu(dfjr(augh,aifs),ehnq)),edeb(enfe(cugq,bzkr(dzdl,edq(dpkz(eaap(bnlv,cahc),dwwb(deht,bifr(eipc,cwwo))),cfkn))),bbga)),abbb),efnd),brrw))),cxsv))),dshn)))),adcz(banw,crcd(bmai,boqy)))),eaos(dzob(acxt,bsxc(dxab(utk(dxra,xjj(dvhg,eezh)),bck(ckvd,cmru)),asou)),cddk))),awme),citp),caer),nce))))))
plus(b(f(minus,minus),minus(pow(f(b,y),mult(pow,mult(minus,a(x,y(plus(pow(baz(x,f),plus),mult(x(plus(a(x,x(pow,y(mult(plus(plus(pow,pow),b(a,pow)),plus),pow))),y),x(y,x(minus(pow,b(a,pow)),a))),f)),a(b(pow(pow(pow(b(f(mult(minus(pow(mult,baz),a(minus(a(pow(minus(a(a,b),x(plus,minus)),mult),a(mult(minus(b,baz(b(plus,y(f(x,y),a(mult,a(a,plus(pow,y(minus(y(pow,pow(x,b(mult,minus))),y(pow,a)),pow)))))),f(x(a(a(pow,baz(baz(minus,pow),f(mult,pow))),x(f,minus(x(baz,baz),mult))),minus),pow(x(b,x(a(b,a),a)),mult(f(baz,a),plus(b,mult(plus,minus(f,x(minus(y(f,mult),y(x,pow(a(x,y(a,y(plus(y,a),a))),y))),plus))))))))),minus),y)),a(x(x,a),baz(x,plus))),y(a(mult,mult(plus,minus)),f))),baz),b(f,baz(b,a))),plus(y,minus)),b(x(pow(mult(a,f),x),mult),f(plus,plus(f,b(pow,plus(x,f(y,minus(y(a(x,x),pow(x(minus(mult(baz,minus),f),x(f(y,minus),mult(y(b,pow),x(x,baz)))),plus)),minus)))))))),plus),baz(x,a)),mult(a(baz(pow,baz),baz(y,a)),baz)),pow(plus,plus))))))),plus)),minus(mult,a(plus(f,y(y(baz,x),x)),pow)))
aqyt(haz,azpk(ayek(tbh(eqji,bihk(clow(acmi,bfsu),hmh(eddh(dfbn(bfps,egzc),bkpm(btkd(eqrb,dcoh),cnx(ehtq,acb))),ehsp))),bbza),cazu(alvn(zlv,dumc(dqqh(acdw,cqit),civo(aqrb(rlk(axma,banh),ccmj),bhvv))),dyvt(hnt,bgys(ddzs(dsrl,dkzu(aibs,drpi(vgf(wxb,botw),dkpf))),phh(aqrr(rkb(eieu(alze,dopm),rw),mct(rfs(tja(dnee(dmpp(ibd,cfpf(bzsk,elcw(biih(bpxl(gwf(ddya,abaq),ekxu(pwd,eexd(baid(bxkh(ccmy(aues,dcae),cvoo(dxht,brgz(aoko,xjk))),pgw(djib(bmwk,ckex),cawr)),cbly))),oxj),ejwq(eoon,dgsb)))),cnnh),eggr(cbsm,eezm)),cuve(bsga(caya,eads),cpxw(azaw(dyne,vxu(atdc(yvu,dnxv(ecjn,pvc)),dcam(byto(dznb,dfaq),covi))),akze(cyqs(aejg,bcff(caom,datx)),dqrm(aybk(gkt(tvx,ca),mxw),adxi(eju(bbli(aldu,aior),epuu),loa)))))),spa(aklj(dxfg(dply,anr),amdo(asmz(enjz(cllx,bpvz(ehxk(adkw(ceid,dwiq(anxe,aoyf)),paq(eqhn(gne(cbxu,cgtv),ylh(chmx,zzz)),bqes(blae,bbmj(dcxa(altd(atcg,blde),ansv),diyb)))),dgsa)),bwbd),eiht)),bggc(cnyb(djub,aswp),cmct(cehw,cfer))))),eoty(bsko(cpga,eduk(bif,adui)),dcud(ekkg,durn(epgk,awsj)))))))))
plus(y(f,baz),baz(pow,plus(mult(baz(mult,f(x,x(f,mult(minus(mult,minus(y(plus,b(plus(x(f(plus(baz,f),f(plus(plus,baz(mult,f)),minus)),mult(y,pow(pow,b(f,f)))),plus(a(a(baz,pow(pow,pow(a(baz(y(baz(b,b(a,x)),f(mult,b)),pow(plus(mult(minus,a),f(plus(b,mult(f(minus,a),x)),mult(minus,y(y(b(y,plus),f(y(x,baz(mult,pow(a(y,minus),f(pow(minus(mult,a),x),f(y,f(baz(minus,plus),b)))))),baz)),plus(mult(x(pow(baz,b(y,f(baz(a(pow(b,mult),pow),a),mult))),y),mult(minus(minus(b,y),plus),f)),plus(y(pow(pow(minus(plus,x),minus(mult,pow(plus(mult(plus,plus),baz),y(baz,a(x(b,baz(baz,baz)),f))))),y(a,mult)),mult),baz)))))),minus(f(minus(pow(b(a,mult(y(a,f(y,minus)),a(minus,b))),minus),f),plus),a))),pow(plus,plus(mult,minus))),y(a,plus(plus(a,pow),b(baz,minus)))))),baz(x,f(plus,minus(f(b,a),pow(a(plus(a(pow,minus),x),x),baz))))),pow(mult,minus(f,pow(a,pow(y(pow,y),baz)))))),x)),baz(pow,x(y,baz(a(x(plus,mult),f(x,plus(f,minus))),minus))))),b)))),y(y(minus,a(plus(y(a(baz(plus,y),pow(f,b)),x),f),pow)),a)),b)))
