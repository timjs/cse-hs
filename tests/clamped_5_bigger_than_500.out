drpj(djyb(voc(cyqo(ceyo(awcb,bcuf),dfap(cyeo(edws(bwfk(dbru(cm(egvd,ctd(cnkx(bcv,bwmf),bgpm)),egrs(auwt,saa)),suv(buow(aovq,czqa(sfl(eto,cbda),aksb)),epmn)),cst(brbm,awoe)),aopa),akoh)),abar),aabl(exd,apem)),amma(cjwz(dlab(biyx(yey,azin(dtwu(bqfe(dctp(ecrq(doir,axgu),arcc(vpg(bwsb(cqab,bsvv),awwg),ojp)),dcpl(afla,aazx(drrm(ebzm(btyz(cgme(assm(beeu,astk),dbqg),ehzx),bgyx),bopd(ampg(eef(auzd,aaqq(bmee,dyoj(mcn,aimn))),aqnp(cspy(admh(bdft,dfyk(clv,dobh(dmgy,dqiv))),dljz(codv(btns(ekyf(ayvh,cjhg),cxor),buya),jf)),cbrh)),axmu(dxhx,eixp(ehwy,bvph(amkj(dgza(admn(arqk,ctuz(ejji(owd(adye,doei(bmkt,csay)),ghu(dyob(imv(yjv(aik(nmj,dkut(bumh(cxsh(birg(awin(dcxe,dxcp(ejxv,akzr)),asew),avnp),oxq),bnni)),afae(cysy,ccvy(delw(aszl,atwx),elmu(dodg,cwoa(xdc(caob(yja,ahs(bgyj,ger)),efji(bdjo(cdea,akyu),mkc)),bgwt(ccat(pse,eprv(ciqr,btz(dopq,cpfi(clyl,bchk)))),xda)))))),djyx(upo,dxyw)),dnzp(abik,hon(doji,apyg))),brni)),bsro(bddc,pme(dfnr,axtq)))),brjg),sp),eftq))))),edhh))),dkkp),axuh)),czaq),aadv),dqsw))
aovw(chnm(efcw,bxlu),bocj(qff(crfs,dqli(cvjg(por(uwk(dcpw,btpd),ecx),jka(ybv,cmmo)),cxdr(bqml(mki(ywy,drey),yzt(awbu(ceif,eqas(lzj(afqr(axdr(bdfs,bdzv),caob),hmt),aoos)),bjpb)),ayny))),bmgi(avqr(anuf,czge(dvmd(dxof,dudd),eewu)),xhd(hjc(clol(dasa,bdja),bmki(gna(bruw(auyd(djug(bokw(yyl,ahmf),zcg(dhcg(cwiu,awyg),clib(dizj(bwh(dnlr,dpoy(dke(alpl(alvv,eaot),dauh),dffv)),eetl),elza))),dkzj(egtp(bktb,elcw),chbx)),acpk),amko(bgoi,dnhg)),eks)),cotn(cobb,eepu(dzaj(njo(eppv(ujg(utq(dojx(dqws(cveo,tcw(ckpf,cvcg(slo(dkum,aiot),pfh))),cpnd),txb),bzyc(ekr(dfdi(arww,ejnw),boay(bcff(bela,fzp(enha,bfrx(aecz(aaao,adyp(nlw,bmqz)),egjz(dtst(dkkd,agtq(akbx,cyh(ifc(cvzi,edjk(cljk(chzb,aksq),brqs(dlfa(adfr(elv(clp(hdg,epmu(dfjr(augh,aifs),ehnq)),edeb(enfe(cugq,bzkr(dzdl,edq(dpkz(eaap(bnlv,cahc),dwwb(deht,bifr(eipc,cwwo))),cfkn))),bbga)),abbb),efnd),brrw))),cxsv))),dshn)))),adcz(banw,crcd(bmai,boqy)))),eaos(dzob(acxt,bsxc(dxab(utk(dxra,xjj(dvhg,eezh)),bck(ckvd,cmru)),asou)),cddk))),awme),citp),caer),nce))))))
plus(b(f(minus,4),minus(pow(f(b,y),mult(pow,mult(4,a(x,y(plus(pow(baz(14,f),plus),mult(x(plus(a(14,x(11,y(mult(plus(plus(11,11),b(a,11)),20),11))),9),x(9,x(minus(11,30),31))),19)),a(b(pow(pow(pow(b(f(mult(minus(pow(mult,baz),a(minus(a(pow(minus(a(31,8),x(20,4)),45),a(mult(minus(8,baz(b(20,y(f(14,9),a(45,a(31,plus(11,y(minus(y(11,pow(14,b(45,4))),y(11,31)),11)))))),f(x(a(a(11,baz(baz(4,11),f(45,11))),x(19,minus(x(46,46),45))),4),pow(x(8,x(a(8,31),31)),mult(f(46,31),plus(8,mult(20,minus(19,x(minus(y(19,45),y(14,pow(a(14,y(31,y(plus(9,31),31))),9))),20))))))))),4),9)),a(x(14,31),baz(14,20))),y(a(45,mult(20,4)),19))),46),b(19,baz(8,31))),plus(9,4)),b(x(pow(mult(31,19),14),45),f(20,plus(19,b(11,plus(14,f(9,minus(y(a(14,14),pow(x(minus(mult(46,4),19),x(f(9,4),mult(y(8,11),x(14,46)))),20)),4)))))))),20),baz(14,31)),mult(a(baz(11,46),baz(9,31)),46)),pow(20,20))))))),20)),minus(45,a(plus(19,y(y(46,14),14)),11)))
aqyt(haz,azpk(ayek(tbh(eqji,bihk(clow(acmi,bfsu),hmh(eddh(dfbn(bfps,egzc),bkpm(btkd(eqrb,dcoh),cnx(ehtq,acb))),ehsp))),bbza),cazu(alvn(zlv,dumc(dqqh(acdw,cqit),civo(aqrb(rlk(axma,banh),ccmj),bhvv))),dyvt(hnt,bgys(ddzs(dsrl,dkzu(aibs,drpi(vgf(wxb,botw),dkpf))),phh(aqrr(rkb(eieu(alze,dopm),rw),mct(rfs(tja(dnee(dmpp(ibd,cfpf(bzsk,elcw(biih(bpxl(gwf(ddya,abaq),ekxu(pwd,eexd(baid(bxkh(ccmy(aues,dcae),cvoo(dxht,brgz(aoko,xjk))),pgw(djib(bmwk,ckex),cawr)),cbly))),oxj),ejwq(eoon,dgsb)))),cnnh),eggr(cbsm,eezm)),cuve(bsga(caya,eads),cpxw(azaw(dyne,vxu(atdc(yvu,dnxv(ecjn,pvc)),dcam(byto(dznb,dfaq),covi))),akze(cyqs(aejg,bcff(caom,datx)),dqrm(aybk(gkt(tvx,ca),mxw),adxi(eju(bbli(aldu,aior),epuu),loa)))))),spa(aklj(dxfg(dply,anr),amdo(asmz(enjz(cllx,bpvz(ehxk(adkw(ceid,dwiq(anxe,aoyf)),paq(eqhn(gne(cbxu,cgtv),ylh(chmx,zzz)),bqes(blae,bbmj(dcxa(altd(atcg,blde),ansv),diyb)))),dgsa)),bwbd),eiht)),bggc(cnyb(djub,aswp),cmct(cehw,cfer))))),eoty(bsko(cpga,eduk(bif,adui)),dcud(ekkg,durn(epgk,awsj)))))))))
plus(y(f,baz),baz(pow,plus(mult(baz(mult,f(x,x(3,mult(minus(10,minus(y(plus,b(plus(x(f(plus(4,3),f(plus(18,baz(10,3)),minus)),mult(y,pow(6,b(3,3)))),plus(a(a(4,pow(6,pow(a(baz(y(baz(b,b(a,12)),f(10,41)),pow(plus(mult(27,43),f(plus(41,mult(f(27,43),12)),mult(27,y(y(b(29,18),f(y(12,baz(10,pow(a(29,27),f(pow(minus(10,43),12),f(29,f(baz(27,18),41)))))),4)),plus(mult(x(pow(4,b(29,f(baz(a(pow(41,10),6),43),10))),29),mult(minus(minus(41,29),18),3)),plus(y(pow(pow(minus(18,12),minus(10,pow(plus(mult(18,18),4),y(4,a(x(41,baz(4,4)),3))))),y(43,10)),10),4)))))),minus(f(minus(pow(b(43,mult(y(43,f(29,27)),a(27,41))),27),3),18),43))),pow(18,plus(10,27))),y(43,plus(plus(43,6),b(4,27)))))),baz(12,f(18,minus(f(41,43),pow(a(plus(a(6,27),12),12),4))))),pow(10,minus(3,pow(43,pow(y(6,29),4)))))),12)),baz(6,x(29,baz(a(x(18,10),f(12,plus(3,27))),27))))),41)))),y(y(27,a(plus(y(a(baz(18,29),pow(3,41)),12),3),6)),43)),41)))
